server/image_status.go

```diff
@@ -67,7 +67,7 @@ func (s *Server) ImageStatus(ctx context.Context, req *pb.ImageStatusRequest) (r
 			},
 		}
 		if req.Verbose {
-			info, err := createImageInfo(status)
+			info, err := createImageInfo(status, s)
```

Comment:
I'd say, instead of passing server in, let's make `createImageInfo` a member function of server